### PR TITLE
[CLOUD-3512] Update to EAP 7.3.0 SP01

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.redhat.com/browse/CLOUD-3512
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -36,11 +36,11 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_730_CR4
+                  ref: EAP_730_SP01
```
Comment (jmesnil) [inline image.yaml:39]: for reference, `EAP_730_SP01` & `EAP_730_CR4` are identical: https://github.com/jboss-container-images/jboss-eap-modules/compare/EAP_730_CR4...EAP_730_SP01

File: image.yaml [hunk 2]
```diff
@@ -36,11 +36,11 @@ modules:
...
           - name: jboss-eap-7-image
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: EAP_730_CR4
+                  ref: EAP_730_SP01
```
Comment (jmesnil) [inline image.yaml:43]: @luck3y tag is missing: https://github.com/jboss-container-images/jboss-eap-7-image/releases/tag/EAP_730_SP01

I've merged https://github.com/jboss-container-images/jboss-eap-7-image/pull/181 so you can tag it now.

**Review by jmesnil — CHANGES_REQUESTED** [review]

**Comment by jmesnil** [comment]
@luck3y LGTM, thanks
I'm holding off merging until https://github.com/jboss-container-images/jboss-eap-7-image.git is tagged.

**Comment by luck3y** [comment]
@jmesnil jboss-eap-7-image has the EAP_730_SP01 tag now.

**Review by jmesnil — APPROVED** [review]

**Comment by jmesnil** [comment]
@luck3y thanks

**Pull request merged**